tinodesdk/src/main/java/co/tinode/tinodesdk/Topic.java

```diff
@@ -1074,7 +1076,7 @@ public PromisedReply<ServerMessage> publish(final Drafty content, final Map<Stri
             if (extraHeaders != null) {
                 head.putAll(extraHeaders);
             }
-            if (!content.isPlain()) {
+            if (!content.isPlain() && !Tinode.VIDEO_CALL_MIME.equals(head.get("mime"))) {
```

Comment:
`Tinode.VIDEO_CALL_MIME.equals(head.get("mime")` should not be needed.

Reply:
Done.